lib/commands/update.ts

```diff
@@ -15,49 +15,34 @@ export class UpdateCommand implements ICommand {
 		this.$projectData.initializeProjectData();
 	}
 
+	static readonly folders: string[] = ["lib", "hooks", "platforms", "node_modules"];
```

Comment:
you have constants required anyway, please add `lib` and `hooks` there and replace all strings with constants